src/ot_orchestration/dags/config/gwas_catalog_sumstats_susie_clumping.yaml

```diff
@@ -0,0 +1,45 @@
+dataproc:
+  python_main_module: gs://genetics_etl_python_playground/initialisation/gentropy/dsdo_top_hits_step/cli.py
```

Comment:
Very minor, but I would change all branch specific reference to `dev`.

Reply:
you are right, thanks!